samples/databases/wide-world-importers/wwi-dw-ssdt/README.md

```diff
@@ -0,0 +1,80 @@
+# WideWorldImportersDW OLAP Database
+
+The Visual Studio SQL Server Data Tools project in this folder is used to construct the WideWorldImportersDW database from scratch on SQL Server or Azure SQL Database. It is possible to vary the data size.
+
+A pre-created version of the database is available for download as part of the latest release of the sample: [wide-world-importers-release](http://go.microsoft.com/fwlink/?LinkID=800630).
+
+### Contents
+
+[About this sample](#about-this-sample)<br/>
+[Before you begin](#before-you-begin)<br/>
+[Run this sample](#run-this-sample)<br/>
+[Disclaimers](#disclaimers)<br/>
+
+
+<a name=about-this-sample></a>
+
+## About this sample
+
+<!-- Delete the ones that don't apply -->
+1. **Applies to:** SQL Server 2016 (or higher), Azure SQL Database
+1. **Key features:** Core database features
+1. **Workload:** OLTP
+1. **Programming Language:** Transact-SQL
+1. **Authors:** Greg Low, Denzil Ribeiro, Jos de Bruijn, Robert Cain
+
+The instructions below are for creating the sample database from scratch.
+
+A pre-created version of the database is available for download as part of the latest release of the sample: [wide-world-importers-release](http://go.microsoft.com/fwlink/?LinkID=800630).
+
+<a name=before-you-begin></a>
+
+## Before you begin
+
+To run this sample, you need the following prerequisites.
+
+**Software prerequisites:**
+
+1. SQL Server 2016 SP1 (or higher) or an Azure SQL Database. Also works with SQL Server 2016 RTM, for Evaluation, Developer, and Enterprise edition.
```

Comment:
Will any SLO of Azure SQL Database do? I am guessing not since there's In-Memory OLTP involved. In that case we should be specific?